prawns/staking/schema.graphql

```diff
@@ -31,9 +32,10 @@ type SubstrateStakingValidatorAccount @entity {
   "address"
   account: String! @index
```

Comment:
the `id` field can be the account now, it doesn't need to be concatenated with the token symbol anymore due to the single network deployments

Reply:
Do we need the `network: SubstrateNetwork! @index` then in all our schemas? 

I guess not as well :D